snmp/metadata.csv

```diff
@@ -1,2 +1,220 @@
+snmp.tcpRetransSegs,count,,segment,,"[Generic router] The total number of segments retransmitted; that is, the number of TCP segments transmitted containing one or more previously transmitted octets.",0,snmp,snmp_snmp.tcpRetransSegs
+snmp.tcpInErrs,count,,segment,,"[Generic router] The total number of segments received in error (e.g., bad TCP checksums).",0,snmp,snmp_snmp.tcpInErrs
+snmp.tcpOutRsts,count,,segment,,[Generic router] The number of TCP segments sent containing the RST flag.,0,snmp,snmp_snmp.tcpOutRsts
+snmp.udpHCInDatagrams,count,,datagram,,"[Generic router] The total number of UDP datagrams delivered to UDP users, for devices that can receive more than 1 million UDP datagrams per second.",0,snmp,snmp_snmp.udpHCInDatagrams
+snmp.udpNoPorts,count,,datagram,,[Generic router] The total number of received UDP datagrams for which there was no application at the destination port.,0,snmp,snmp_snmp.udpNoPorts
+snmp.udpInErrors,count,,datagram,,[Generic router] The number of received UDP datagrams that could not be delivered for reasons other than the lack of an application at the destination port.,0,snmp,snmp_snmp.udpInErrors
+snmp.udpHCOutDatagrams,count,,datagram,,"[Generic router] The total number of UDP datagrams sent from this entity, for devices that can transmit more than 1 million UDP datagrams per second.",0,snmp,snmp_snmp.udpHCOutDatagrams
+snmp.sysStatMemoryTotal,gauge,,byte,,[F5 BIG-IP] The total memory available in bytes for TMM (Traffic Management Module).,0,snmp,snmp_snmp.sysStatMemoryTotal
+snmp.sysStatMemoryUsed,gauge,,byte,,[F5 BIG-IP] The memory in use in bytes for TMM (Traffic Management Module).,0,snmp,snmp_snmp.sysStatMemoryUsed
+snmp.sysGlobalTmmStatMemoryTotal,gauge,,byte,,[F5 BIG-IP] The total memory available in bytes for TMM (Traffic Management Module).,0,snmp,snmp_snmp.sysGlobalTmmStatMemoryTotal
+snmp.sysGlobalTmmStatMemoryUsed,gauge,,byte,,[F5 BIG-IP] The memory in use in bytes for TMM (Traffic Management Module).,0,snmp,snmp_snmp.sysGlobalTmmStatMemoryUsed
+snmp.sysGlobalHostOtherMemoryTotal,gauge,,byte,,[F5 BIG-IP] The total other non-TMM memory in bytes for the system.,0,snmp,snmp_snmp.sysGlobalHostOtherMemoryTotal
+snmp.sysGlobalHostOtherMemoryUsed,gauge,,byte,,[F5 BIG-IP] The other non-TMM memory in bytes currently in use for the system.,0,snmp,snmp_snmp.sysGlobalHostOtherMemoryUsed
+snmp.sysGlobalHostSwapTotal,gauge,,byte,,[F5 BIG-IP] The total swap in bytes for the system.,0,snmp,snmp_snmp.sysGlobalHostSwapTotal
+snmp.sysGlobalHostSwapUsed,gauge,,byte,,[F5 BIG-IP] The swap in bytes currently in use for the system.,0,snmp,snmp_snmp.sysGlobalHostSwapUsed
+snmp.sysMultiHostCpuTable,gauge,,,,[F5 BIG-IP] A table containing entries of system CPU usage information for a system.,0,snmp,snmp_snmp.sysMultiHostCpuTable
```

Comment:
I think this was a typo originally -- we should remove this as it isn't a metric, rather the root of where some metrics will come from